Jellyfin/dist/presence.js

```diff
@@ -0,0 +1,459 @@
+// official website
+const JELLYFIN_URL = "jellyfin.org";
+
+// web client document.title value (case sensitive)
+const DOCUMENT_TITLE = "Jellyfin";
+
+// web client app name
+const APP_NAME = "Jellyfin Web";
+
+// all the presence art assets uploaded to discord
+const PRESENCE_ART_ASSETS = {
+    download: "downloading",
+    live: "live",
+    logo: "banner-icon",
+    pause: "pause",
+    play: "play",
+    read: "reading",
+    search: "search",
+    write: "writing"
+}
+
+let presence;
+
+async function init() {
+    let validPage = false;
+
+    // jellyfin website
+    if (location.host === JELLYFIN_URL) {
+        validPage = true;
+        log("Jellyfin website detected");
+
+    // web client
+    } else {
+        try {
+            let data = JSON.parse(localStorage.getItem("jellyfin_credentials"));
+
+            for (let server of data.Servers) {
+                // user has accessed in the last 30 seconds, should be enough for slow connections
+                if (Date.now() - new Date(server.DateLastAccessed) < 30 * 1000) {
+                    validPage = true;
+                    log("Jellyfin web client detected");
+                }
+            }
+        } catch (e) {
+            validPage = false;
+        }
+    }
+
+    if (validPage) {
+        presence = new Presence({
+            clientId: "669359568391766018",
+            mediaKeys: false
+        });
+
+        presence.on("UpdateData", updateData);
+    }
+}
+init();
+
+// View https://docs.premid.app/en/dev/presence/class -> presenceData interfac
+let presenceData = {
+    largeImageKey: PRESENCE_ART_ASSETS.logo,
+    // smallImageKey: "key",
+    // smallImageText: "Some hover text",
+    // details: "",
+    // state: "",
+    // startTimestamp: null,
+    // endTimestamp: 1577151472000
```

Comment:
remove comma after large image key, and remove the unneeded comments.

Reply:
Removed comments in commit [2cbff2aadb8940809157cd9169073603a8124af7](https://github.com/alexbcberio/Presences/commit/2cbff2aadb8940809157cd9169073603a8124af7)